src/main/java/teammates/ui/webapi/action/GetCourseAction.java

```diff
@@ -17,7 +20,6 @@ protected AuthType getMinAuthLevel() {
     public void checkSpecificAccessControl() {
         String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
 
-        // TODO enable access for student
```

Comment:
Should we enable access for student here? Perhaps need to ask core members

Reply:
Unfortunately I've spoken with Xiao Pu and he says that since student will return a list of courses that the student has access to, that focuses more on GET /courses instead of GET /course.

Reply:
So as of now, a student cannot access this endpoint is it? 

Right now, Gatekeeper will throw an error if no instructor is found from the id.

Reply:
Yup! The action has been moved to /courses.